src/crypto/cn/CryptoNight_x86.h

```diff
@@ -679,7 +679,7 @@ inline void cryptonight_single_hash(const uint8_t *__restrict__ input, size_t si
     }
 
     for (size_t i = 0; i < props.iterations(); i++) {
-        __m128i cx;
+        __m128i cx = _mm_setzero_si128();
```

Comment:
No. Don't add stuff to the main loop because some program says so. It was carefully crafted to run as fast as possible.